test/e2e/hvpa_crd_test.go

```diff
@@ -0,0 +1,51 @@
+package e2e_test
+
+import (
+	"bytes"
+	"context"
+	_ "embed"
+	"fmt"
+
+	"k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1beta1"
+	"k8s.io/apimachinery/pkg/util/yaml"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/controller/controllerutil"
+
+	"github.com/gardener/virtual-garden/pkg/virtualgarden"
+)
+
+//go:embed resources/hvpa.yaml
+var hvpaCrd []byte
+
+// deployHVPACRD deploys the HVPA CRD.
+func deployHVPACRD(ctx context.Context, c client.Client) error {
```

Comment:
Is this code still required? I thought you would only check for the existence of the hvpa CRD, not deploying it?

Reply:
It is needed to prepare the integration test (test/e2e/e2e_test.go, BeforeSuite).

The installation that is being tested in the integration test does not deploy the crd, but it assumes that the the crd is there. To fulfil this assumption, we must deploy the crd in BeforeSuite.

Reply:
okay gotcha. thanks